api/envoy/extensions/transport_sockets/tls/v3/common.proto

```diff
@@ -66,25 +66,17 @@ message TlsParameters {
   //
   //   [ECDHE-ECDSA-AES128-GCM-SHA256|ECDHE-ECDSA-CHACHA20-POLY1305]
   //   [ECDHE-RSA-AES128-GCM-SHA256|ECDHE-RSA-CHACHA20-POLY1305]
-  //   ECDHE-ECDSA-AES128-SHA
```

Comment:
I am curious where we can check it is enabled by default, do we have tests for it?

Reply:
Hi @daixiang0  There are test cases for the cipher suites in https://github.com/envoyproxy/envoy/blob/main/test/extensions/transport_sockets/tls/ssl_socket_test.cc

Reply:
Good, lgtm.